domains/samuelcodez.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "SamuelCodez",
+    "email": "samuelcodez@outlook.com"
+  },
+  "record": {
+    "CNAME": "https://samuelcodez.github.io"
```

Comment:
CNAME has to be a hostname and not a URL. If you wanted a URL redirection, change CNAME to URL. If you want a DNS record, remove the `https://`